### ISSUE TITLE
Illegal punning

### ISSUE THREAD

**Issue by mereolog** [edmcouncil/auto#42]
Ontology uses the [illegal punning, where one IRI is used for an ObjectProperty and DataProperty](https://www.w3.org/TR/owl2-new-features/#F12:_Punning)- see the warnings below: 

```
WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/model in punning not allowed [ObjectProperty: model, DataProperty: model]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/track in punning not allowed [ObjectProperty: track, DataProperty: track]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/row3PosCount in punning not allowed [ObjectProperty: row3PosCount, DataProperty: row3PosCount]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/wheelDiameter in punning not allowed [DataProperty: wheelDiameter, ObjectProperty: wheelDiameter]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/brand in punning not allowed [ObjectProperty: brand, DataProperty: brand]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/height in punning not allowed [ObjectProperty: height, DataProperty: height]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/fuelType in punning not allowed [DataProperty: fuelType, ObjectProperty: fuelType]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/row5PosCount in punning not allowed [ObjectProperty: row5PosCount, DataProperty: row5PosCount]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/refuelPosition in punning not allowed [ObjectProperty: refuelPosition, DataProperty: refuelPosition]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/row4PosCount in punning not allowed [ObjectProperty: row4PosCount, DataProperty: row4PosCount]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/bodyType in punning not allowed [DataProperty: bodyType, ObjectProperty: bodyType]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/tireWidth in punning not allowed [ObjectProperty: tireWidth, DataProperty: tireWidth]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/tankCapacity in punning not allowed [DataProperty: tankCapacity, ObjectProperty: tankCapacity]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/rowCount in punning not allowed [ObjectProperty: rowCount, DataProperty: rowCount]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/doorCount in punning not allowed [ObjectProperty: doorCount, DataProperty: doorCount]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/year in punning not allowed [ObjectProperty: year, DataProperty: year]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/displacement in punning not allowed [DataProperty: displacement, ObjectProperty: displacement]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/steeringWheelSide in punning not allowed [DataProperty: steeringWheelSide, ObjectProperty: steeringWheelSide]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/row1PosCount in punning not allowed [DataProperty: row1PosCount, ObjectProperty: row1PosCount]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/row2PosCount in punning not allowed [DataProperty: row2PosCount, ObjectProperty: row2PosCount]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/driveType in punning not allowed [ObjectProperty: driveType, DataProperty: driveType]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/towWeight in punning not allowed [DataProperty: towWeight, ObjectProperty: towWeight]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/wheelbase in punning not allowed [DataProperty: wheelbase, ObjectProperty: wheelbase]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/wmi in punning not allowed [DataProperty: wmi, ObjectProperty: wmi]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/length in punning not allowed [DataProperty: length, ObjectProperty: length]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/transmissionType in punning not allowed [DataProperty: transmissionType, ObjectProperty: transmissionType]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/vin in punning not allowed [ObjectProperty: vin, DataProperty: vin]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/axleCount in punning not allowed [DataProperty: axleCount, ObjectProperty: axleCount]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/engineConfiguration in punning not allowed [ObjectProperty: engineConfiguration, DataProperty: engineConfiguration]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/maxTorque in punning not allowed [DataProperty: maxTorque, ObjectProperty: maxTorque]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/maxPower in punning not allowed [ObjectProperty: maxPower, DataProperty: maxPower]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/wheelWidth in punning not allowed [ObjectProperty: wheelWidth, DataProperty: wheelWidth]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/driverPostion in punning not allowed [DataProperty: driverPostion, ObjectProperty: driverPostion]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/width in punning not allowed [DataProperty: width, ObjectProperty: width]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/wheelCount in punning not allowed [DataProperty: wheelCount, ObjectProperty: wheelCount]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/grossWeight in punning not allowed [ObjectProperty: grossWeight, DataProperty: grossWeight]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/curbWeight in punning not allowed [DataProperty: curbWeight, ObjectProperty: curbWeight]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/gearCount in punning not allowed [DataProperty: gearCount, ObjectProperty: gearCount]
   WARN  18:00:21  Illegal redeclarations of entities: reuse of entity https://spec.edmcouncil.org/auto/ontology/VS/VehicleSignals/tireDiameter in punning not allowed [ObjectProperty: tireDiameter, DataProperty: tireDiameter]
```
For the explanation why this kind of punning should be avoided see https://pure.uva.nl/ws/files/763164/68628_06.pdf page 59: "Punning enables straightforward meta modelling without the impact on decidability that RDF and OWL Full modelling have. Because of interaction with the semantics of OWL Full, punning is not allowed between data and object properties."

**Comment by trypuz** [comment]
@mereolog, most of the properties listed above are already in VehicleCore. So we're going to remove them from the vehicle signal ontology.

**Comment by mereolog** [comment]
This is now solved - no punning in d575e76e2993e064c3ae2b40a47a008ac2045ae4
